app/models/like.rb

```diff
@@ -1,4 +1,7 @@
 class Like < ApplicationRecord
-  belongs_to :author, class_name: 'User'
+  belongs_to :user
```

Comment:
For the associations to fit the ERD given, you should use `:author` instead of `:user`: 
```
belongs_to :author, class_name: 'User'
```